### PR TITLE
Add debug logs for hub encryption flaky issue

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Getting hub encryption test results is slow when it is in hosted mode. This PR will solve the problem.
 I tested over 5 times. No issues anymore.
Ref: https://issues.redhat.com/browse/ACM-11497

### CODE REVIEW

File: test/e2e/hub_templates_encryption_test.go [hunk 1]
```diff
@@ -260,3 +266,41 @@ var _ = Describe("Test Hub Template Encryption", Ordered, func() {
...
+func DebugHubEncryption(policyName, secretName, configMapName, hubNs, clusterNamespaceOnHub string) {
+	By("Collecting debug information")
+
+	dumpDebug := func(title string, output string) {
+		GinkgoWriter.Printf("\n=== DEBUG: %s:\n%s", title, output)
+	}
+
+	outYaml := "-o=yaml"
+
+	debugHubCmds := map[string][]string{
```
[the remark this answers is not in the document]
Comment (mprahl) [reply test/e2e/hub_templates_encryption_test.go:279]: The secret is also on the managed cluster so it'd be nice to have that command also on the managed side.

[the remark this answers is not in the document]
Comment (yiraeChristineKim) [reply test/e2e/hub_templates_encryption_test.go:282]: Thanks!

**Review by mprahl — APPROVED** [review]

**Pull request merged**